sdk/trace/benchmark_test.go

```diff
@@ -117,21 +117,21 @@ func BenchmarkSpanWithAttributes_all_2x(b *testing.B) {
 
 func BenchmarkTraceID_DotString(b *testing.B) {
 	t, _ := trace.TraceIDFromHex("0000000000000001000000000000002a")
-	sc := trace.SpanContext{TraceID: t}
+	sc := trace.SpanContext{}.WithTraceID(t)
```

Comment:
This seems error prone if the `NewSpanContext` constructor ever is updated to include any state initialization. Should it be switched to use that instead?